e2e/playwright/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "plugin-e2e-tests",
+  "scripts": {
+    "test": "PW_SLOMO=200 npm run test --prefix ../../../mattermost-server/e2e/playwright -- --project=chrome --config='../../../mattermost-plugin-github/e2e/playwright/playwright.config.ts'",
```

Comment:
Can the path to `mattermost-plugin-github` not be hard coded? E.g. can we use `pwd` here?

Reply:
@hanzei I was able to get this to work with:

```sh
export SERVER_PLAYWRIGHT_DIR=../../../mattermost-server/e2e-tests/playwright
export PLUGIN_PLAYWRIGHT_DIR=$(pwd)
```

Then in `package.json`:

```sh
npm run test --prefix $SERVER_PLAYWRIGHT_DIR -- --project=chrome --config=\"${PLUGIN_PLAYWRIGHT_DIR}/playwright.config.ts\"
```

This fixes the issue in `package.json`, but the problem still persists in `tsconfig.json`. I think this can be resolved by using https://www.typescriptlang.org/tsconfig#extends, and having a `gitignore`'d `tsconfig.json` file that extends an existing one, autogenerated from the checked-in one. Then you can edit that as needed, and your git index should stay clean.

I'm going to leave this as-is for now if that is okay, since I would rather solve both issues at once. You can edit these files by hand for now. I've made a ticket to revisit this here https://mattermost.atlassian.net/browse/MM-51863